refactor: streamline proxy routing rules and improve error handling f…

rezatg opened the pull request:
…or active network ports

### Change Description

What does this change add to or fix in the project?

### Checklist

- [ ] Code has been tested.
    - [ ] Windows 10
    - [ ] Windows 11
    - [ ] macOS
    - [ ] Linux
- [ ] Relevant documentation has been updated.

### Related Links

Link to any related issues. Example: Closes #123

ircfspace commented:
Hi,
Thank you for your first PR and for taking the time to contribute to the development of the app.

There are a couple of things that need to be fixed:

1) The two variables [you removed](https://github.com/bepass-org/oblivion-desktop/pull/858/files#diff-a58aff7d99fa59d063cbdad8cf9af6029b59b69927a8ae58c08f6f126bb6a835L16) are causing issues in proxy.ts:
```
Cannot find name 'oldProxyHost'
Cannot find name 'oldProxyPort'
```

2) Oblivion’s slogan, as stated in the main repository, is "Internet, for all or none!" — please make sure to [restore it](https://github.com/bepass-org/oblivion-desktop/pull/858/files#diff-916ac20bc927bc30607a0b7e018a21583afebb0110c7599ca694a452569e0951L230).

rezatg commented:
Hi,
Thank you for your feedback! I’ve addressed the issues you mentioned:

1. Restored the missing variables oldProxyHost and oldProxyPort in proxy.ts to resolve the reference errors.
2. Updated Oblivion’s slogan to "Internet, for all or none!" as stated in the main repository.

Please let me know if there’s anything else that needs adjustment. 😊
